### ISSUE TITLE
DNS Server

### ISSUE THREAD

**Issue by wolfbek** [martinsauter/WLAN-VPN-Pi#3]
Hello I loved you tutorial. I tried one of the guides you mentioned before and did everything step by step but this script made everything very easy. However I still see my ISP's DNS server. How can I easily change to custom DNS servers like Google's or Opennic's ?

Greeting Henok

**Comment by martinsauter** [comment]
Hi Henok, 

hm, thanks for spotting this, I can reproduce the issue here as well. Did you install v1.4 or 1.3 of the package? I don't think this was the case in v1.3. Anyway, a quick fix is to change the second last line in /etc/dhcp/dhcpd.conf 

from
```
option domain-name-servers 192.168.55.1;
```
to
```
option domain-name-servers 8.8.8.8;
```
which is the Google DNS server. After the reboot, the DNS request will then go through the tun0 interface and is no longer visible to the local ISP.

The downside of this approach is that in some countries the Google DNS service is blocked or landing pages in hotels require a redirect by the local DNS. In other words not a clean solution, the DNS requests should go to the local ISP when the tunnel is not in place and through the tunnel once it is in place. 

I'll investigate a bit if I can improve this.

**Comment by martinsauter** [comment]
Fixed with v1.41
